### DISCUSSION TITLE
What's the color property for the "View problem" background?

### DISCUSSION THREAD

**Discussion by slhck** [microsoft/vscode-discussions#1014] [Extension Development QnA] [answered]
Consider this problem "popup" window (I don't know what the right terminology is):

<img width="533" alt="image" src="https://github.com/microsoft/vscode-discussions/assets/582444/2c973278-1cfd-4df4-973b-107570346106">

Here, when writing a dark theme, the white background is obviously a bad choice. How do I override it?

**Comment by r3m0t** [comment, marked as answer]
`editorMarkerNavigation.background` https://code.visualstudio.com/api/references/theme-color#editor-widget-colors

**Reply by slhck** [reply]
Thanks, that does the job!
